src/app/app-routing.module.ts

```diff
@@ -1,11 +1,22 @@
 import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
+import { RouterModule, Routes } from '@angular/router';
+import { ContentComponent } from './content/content.component';
+import { ArticleComponent } from './article/article.component';
+import { CreateArticleComponent } from './create-article/create-article.component';
 
 
-const routes: Routes = [];
+const appRoutes: Routes = [
+    { path:'', component : ContentComponent},
```

Comment:
You need not load the same component for different routes. Instead you can re-use the existing route like below

`{ path:'', component : ContentComponent}` can be replaced with 
```
{ path:'', redirectTo: 'content', pathMatch: 'full' }
```

Reply:
`routing` changes implemented